sdcm/send_email.py

```diff
@@ -180,6 +181,9 @@ def send_report(self, results):
         if email is None:
             self.log.error("Failed to prepare email", exc_info=True)
             return
+        if not self.email_recipients:
```

Comment:
move this code to the beginning of the method, and set log level as warning

Reply:
It's purpose is to be able to test this method, skipping sending part. So it is perfect place for it

Reply:
FIxed.